rebound/librebound.py

```diff
@@ -64,6 +64,14 @@ def additional_forces(self,func):
             # C function pointer
             self.clibrebound.set_additional_forces_with_parameters(func)
 
+    @property
+    def post_timestep_modifications(self):
+        return self.fp   # might not be needed
```

Comment:
Woops, that was a bad one!  I made an additional pointer, and renamed fp for consistency.  In the case we use a C function, I can't see a case where you would want the pointer from within python, but I made a stub anyway...in the case where we use a C function I set these variables to the string "C function pointer value currently not accessible from python.  Edit librebound.py" (I do this also now for additional forces, which didn't have fp set before when passed a C function).

I also made it so you can pass a python function to post_timestep_modifications to parallel what's there for additional_forces